### PR TITLE
Edits to README doc

### PR DESCRIPTION
**Pull request by KarenGivvin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,37 +24,37 @@ Once you enable nbsimplegrader you will see this toolbar at the top of your note
...
 
 *Once you have published a notebook you won't be able to edit the master copy. If you do want to make changes, you need to click the **Unpublish** button at the top. If you unpublish, the notebook will be removed from all student accounts in all of your classes and your master copy will again be editable.*
 
-Note: when a notebook is published to students, a copy — exactly like the copy created for students — is placed in your Classwork folder under each class. You can access this copy from the dashboard under the Jupyter tab of My Coursekata, or by going into your Classwork folder. This is usually the copy you'll want to work in if you bring it up during class.
+When a notebook is published to students, a copy — exactly like the copy created for students — is placed in your Classwork folder on your Jupyter hub. You can also access this copy from the dashboard under the Jupyter tab of My CourseKata. If you work in a notebook during class, you'll usually want to access it from your Classwork folder.
```
Comment (jimstigler) [inline README.md:37]: @KarenGivvin I think these edits make it less clear... you can access the copy either from the dashboard or from inside the classwork folder... both are equally good.

File: README.md [hunk 2]
```diff
@@ -2,17 +2,19 @@
...
 
 ## Getting Started: A Simple Workflow
 
-There is a lot you can do with nbsimplegrader; we will start by introducing a simple workflow to support creating, distributing and collecting notebooks to/from one or more classes of students. Later we will introduce more advanced cell-authoring tools.
+There is a lot you can do with nbsimplegrader; we will start by introducing a simple workflow to support creating, distributing and collecting notebooks to/from one or more classes of students. Following that we will introduce more advanced cell-authoring tools.
 
 ### Enabling the nbsimplegrader extension
 
-To enable the user interface for nbsimplegrader, open your JupyterHub and click the button labelled **nbsimplegrader** in the main toolbar.
+To enable the user interface for nbsimplegrader, open a JupyterHub notebook and click the button labelled **nbsimplegrader** in the main toolbar.
```
Comment (jimstigler) [inline README.md:17]:
```suggestion
To enable the user interface for nbsimplegrader, open a Jupyter notebook from inside the CourseKata JupyterHub and click the button labelled **nbsimplegrader** in the main toolbar.
```

File: README.md (hunk 1 above)
Comment (KarenGivvin) [reply README.md:37]: Point taken.  My revision reflected my understanding of the original, though.  I don't know how to edit my prior edit, but what if the last sentence were to be: 
If you work in a notebook during class, you can access it from either location.

**Comment by jimstigler** [comment]
I think the point was not that you could access it in either way, but that
you should use this copy if you work during class... which I thought was
clear in the original.

**Comment by KarenGivvin** [comment]
OK.  Then I get it.  I'd interpreted "this is the copy..." as being specific to the location indicated immediately prior.  But I see that the location is irrelevant.

File: README.md (hunk 1 above)
Comment (jimstigler) [inline README.md:37]:
```suggestion
When a notebook is published to students, a copy — exactly like the copy created for students — is placed in your Classwork folder on your Jupyter hub. You can also access this copy from the dashboard under the Jupyter tab of My CourseKata. Use this copy if you want to work in the notebook during class.
```

**Pull request merged**